lib/Resque/Log.php

```diff
@@ -10,8 +10,11 @@ class Resque_Log extends Psr\Log\AbstractLogger
 {
 	public $verbose;
 
-	public function __construct($verbose = false) {
+    public $debug;
```

Comment:
Looking at how this is applied, I think this isn't the correct name for this property.

Also, it would probably be better to adjust the existing property `->verbose` instead of introducing a new parameter with non-configurable behavior.

I agree that being able to filter log entries would be immensely helpful for daily use.  What I propose here, then, is to expand the valid values of the `->verbose` property.  Retaining the meanings of `true` and `false` to mean "everything" and "nothing", respectively, we could additionally support passing an array of PSR-3 log level constants/strings.  Any log entry whose level is in the array is displayed; all others are skipped.

One thing to note, however, is that `Resque_Log` isn't meant to be used in most cases as the actual logger implementation, and will likely be dropped in a future release, in favor of a dependency on the PSR-3 virtual package.  It's essentially only included for backwards compatibility while the project moves away from bundling dependencies.  So you may wish to abandon this change entirely, in that light, and instead use a more capable PSR-3 logger implementation.